.github/workflows/preDeploy.yml

```diff
@@ -95,6 +95,9 @@ jobs:
         uses: Expensify/App/.github/actions/composite/setupGitForOSBotify@main
         with:
           GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+          OS_BOTIFY_APP_ID: ${{ secrets.OS_BOTIFY_APP_ID }}
```

Comment:
Does the `APP_ID` and `INSTALLATION_ID` really have to be secret? I don't know the answer, but if these _could_ be just hard-coded in plain text it could make this PR a bit simpler

Reply:
In the example, the APP_ID is marked secret, but not the installation ID. I'm not actually sure if it does need to be this way, so I just stuck with the example

https://github.com/tibdex/github-app-token/blob/main/README.md

Reply:
Looks like we can actually get rid of the installation ID in the latest version of this so I'm going to update it now